config/projects.ts

```diff
@@ -9,8 +9,8 @@ export const projects = [
     links: {
       github: "https://github.com/Rate-Limiting-Nullifier/circom-rln",
       website: "https://rate-limiting-nullifier.github.io/rln-docs/",
-      twitter: "none",
-      discord: "none",
+      twitter: null,
```

Comment:
```suggestion
      twitter: "",
```

```suggestion
      
```

choose one. having null in the midst of strings is a weird choice. This is also a sign that this file needs to be typed.